conf/conf.go

```diff
@@ -86,60 +88,90 @@ func (s *Schedule) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 // Notify is the settings of notification
 type Notify struct {
-	Retry global.Retry `yaml:"retry"`
-	Dry   bool         `yaml:"dry"`
+	Retry global.Retry `yaml:"retry" json:"retry,omitempty" jsonschema:"title=retry,description=the retry settings"`
+	Dry   bool         `yaml:"dry" json:"dry,omitempty" jsonschema:"title=dry,description=if true, the notification will not be sent,default=false"`
```

Comment:
there's a comma in the description, while jsonschema uses commas as delimiters, this results in a wrong description in the generated JSON schema (refer L181 of schema.json).
and there are other cases.